web/frontend/src/lib/components/collapse.svelte

```diff
@@ -36,7 +36,10 @@ onMount(() => {
 });
 </script>
 
-<div class="relative w-full" id={title}>
+<div
+  class="relative w-full"
+  id={encodeURIComponent(title)}
```

Comment:
Any chance that this element id is used elsewhere in app? IIRC, it is bad to have multiple elements with the same ID

Reply:
You are correct, but there shouldn't be uses of the same ID on the control page. On the config page there are conflicts, but this component isn't rendered there.